replica_client.go

```diff
@@ -8,20 +8,35 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"time"
 
 	"github.com/superfly/ltx"
 )
 
 var ErrStopIter = errors.New("stop iterator")
 
+// Metadata keys for storing LTX file timestamps across different storage backends.
+const (
+	// MetadataKeyTimestamp is used by S3 and GCS for object metadata.
+	MetadataKeyTimestamp = "litestream-timestamp"
+
+	// MetadataKeyTimestampAzure is used by Azure Blob Storage (no hyphens, C# identifier rules).
+	MetadataKeyTimestampAzure = "litestreamtimestamp"
+
+	// HeaderKeyTimestamp is used by NATS for object headers.
+	HeaderKeyTimestamp = "Litestream-Timestamp"
+)
```

Comment:
If these are specific to the implementation then I'd personally move them into their respective packages (`s3`, `gs`, etc) even if that means duplicating some.